prime-router/docs/features/0005-managing-translation-schemas.md

```diff
@@ -0,0 +1,793 @@
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Production</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    validating.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    previous-previous-valid-2023-02-25T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+11. The azure function for validation runs successfully
+    <details>
+      <summary>Local</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-03-29T23:53:00.488Z.txt
+                    previous-valid-2023-02-25T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Staging</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-02T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Production</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-03T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+
+#### Validation
+
+The initial validation will mimic how the current integration tests work.  Each translation schema will:
+
+- be checked to make sure it can be successfully
+- ran with its sample input and checked against its sample output failing translation schemas will be indicated in the output
+
```

Comment:
Merge these into one bullet point? They show as separate bullet points.
```suggestion
- be checked to make sure it can be successfully ran with its sample input and checked against its sample output 
- failing translation schemas will be indicated in the output
```